components/vf-link/vf-link.njk

```diff
@@ -1,15 +1,45 @@
-<p class="vf-text">
-  <a class="vf-link" href="#">An inline link</a>
-</p>
+{% spaceless %}
 
-<p class="vf-text">
-  <a class="vf-link vf-link--hover" href="#">An inline link :hover</a>
-</p>
+{# We need to know if links should be disabled #}
+{% set disabled = false %}
 
-<p class="vf-text">
-  <a class="vf-link vf-link--visited" href="#">An inline link :visited</a>
-</p>
+<a
+{# Using an ID? Really?? That goes here #}
+{%- if id %} id="{{-id-}}"{% endif -%}
 
-<p class="vf-text">
-  <a class="vf-link" href="#" disabled>An inline link :disabled</a>
-</p>
+{# Gotta have a link #}
+href="{{-href-}}"
+
+{# Have a target like _blank? That goes here #}
+{%- if target %} target="_{{-target-}}"{% endif -%}
+
+{# Here is where we are adding the vf-text--body modifier #}
+class="vf-link
+
+{% if theme %}vf-link--{{ theme -}}{% endif %}
+
+{# If you're styling the shape of the item, the classnames will render in this for loop #}
+{%- if style -%}
+  {% for styles in style %}
+    {# Link is disabled? #}
+    {%- if styles == "disabled" %}
```

Comment:
I think we need to updated this to output `vf-link--disabled`, if I'm following?

Reply:
this is creating `vf-link--disabled` for me as well as adding the HTML5 'wotsit' of `disabled` to the element too.

Originally it was adding `vf-link--secondary--disabled`, so I moved the logic around a little to stop it from doing that.

Is it not working for you?